src/main/java/mate/academy/spring/model/dto/response/UserResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.model.dto.response;
+
+public class UserResponseDto {
+    private Long id;
+    private String email;
+    private String password;
```

Comment:
```suggestion
```
you don`t need password in response dto class

Reply:
Agree with you, fixed